src/pages/user.tsx

```diff
@@ -62,7 +62,7 @@ export function User() {
   }, [searchTerm, usersData]);
 
   const openModal = () => {
```

Comment:
Suggestion: change the name to something like this: openManageUserModal

Reply:
i did that